msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,165 @@
+
+Before you can create and manage pre-approvals. You need to connect PowerShell to your tenant using Microsoft Graph. You can use the `Connect-MgGraph` cmdlet to do this. You will need to connect with the following permissions to create, manage, and delete pre-approval policies. 
+
+1. `TeamworkAppSettings.ReadWrite.All`
+1. `Policy.ReadWrite.Authorization`
+1. `AppCatalog.Read.All`
+
+```powershell
+connect-MgGraph -Scopes @('TeamworkAppSettings.ReadWrite.All', 'Policy.ReadWrite.Authorization')
+```
+
+# Enable preapproval for RSC permissions
+
+For pre-approval to function you need to change your RSC settings to allow pre-approved apps only. You can do this using the `Set-MgBetaChatRscConfiguration` and `Set-MgBetaTeamRscConfiguration` cmdlets. Below are the available configurations available for these settings.
+
+|Configuration| Description|
+|---|---|
+|`ManagedByMicrosoft`|This is the default state of Teams RSC when you first create your tenant. It is dynamic and can be changed by Microsoft, it is currently default to approve RSC
+|`ApprovedForAllApps`|Any RSC enabled app can be consented to by users in your tenant. With these settings enabled, preapproval of RSC permissions is not needed as all RSC enabled apps are approved for use
+|`ApprovedForPreApprovedAppsOnly`|RSC enabled apps can be consented on an app-by-app basis. Admins can choose which apps are approved for which RSC permissions.
+|`DisabledForAllApps`| No RSC permissions can be consented to by users. In this state, preapproval of RSC permissions will have no effect on users ability to consent to RSC enabled apps
+
+   >[!WARNING]
+   >Changing your Chat RSC or Team RSC configuration to DisabledForAllApps will disable pre-approval in your tenant and users will encounter errors when attempting to install RSC enabled apps.
+
+## Enable preapproval for Chat RSC: 
+You can check and change the current state of your Chat RSC configurations by using `Get-MgBetaChatRscConfiguration` and `Set-MgBetaChatRscConfiguration` cmndlets. 
+
+1. `Get-MgBetaChatRscConfiguration`
+1. `Set-MgBetaChatRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+## Enable preapproval for Team RSC:
+You can check the current state of your Team RSC configurations by using the `Get-MgBetaTeamsRscConfiguration` and `Set-MgBetaTeamRscConfiguration` cmndlets. 
+
+1. `Get-MgBetaTeamRscConfiguration`
+1. `Set-MgBetaTeamRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+# Create a preapproval
+
+Preapprovals can be created to preapprove RSC permissions against certain sensitivity labels in your tenant. This means you can dictate what data RSC-enabled apps have access to. In this section we will create a pre-approval without a sensitivity label for use with any sensitivity group, and with a sensitivity label for use against certain data. To create a pre-approval you will need the following information:
```

Comment:
You should also mention that admins can preapprove admins for specific permissions in addition to specific sensitivity labels. As an example, you can create a policy that allows consent for Files.Read.Group for Teams where sensitivity label is "Confidential"